src/lexer.rs

```diff
@@ -704,17 +704,33 @@ impl<'a> Lexer<'a> {
         '\x20'..='\x26' | '\x28'..='\x5b' | '\x5d'..='\u{10FFFD}' => {
           let _ = self.read_char();
         }
+        // SESC
+        '\\' => {
+          let _ = self.read_char();
+          if let Some(&(_, ch)) = self.peek_char() {
+            match ch {
+              '\x20'..='\x7e' | '\u{0128}'..='\u{10FFFD}' => {
```

Comment:
What is \u{0128}?  I decode that as Ĩ, which does not have a special meaning in CDDL.

Reply:
@cabo And this is why I tagged you :) ... this is a typo ... thanks for catching! will fix